### PR TITLE
Apply minor polish to RESTEasy Reactive extension using Java 11 Collection methods

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java [hunk 1]
```diff
@@ -22,11 +21,11 @@
 
 public class ResteasyReactiveJsonbProcessor {
 
-    private static final List<String> VERTX_SERIALIZERS = Arrays.asList(
+    private static final List<String> VERTX_SERIALIZERS = List.of(
             VertxJson.JsonObjectSerializer.class.getName(),
             VertxJson.JsonArraySerializer.class.getName());
 
-    private static final List<String> VERTX_DESERIALIZERS = Arrays.asList(
+    private static final List<String> VERTX_DESERIALIZERS = List.of(
```
Comment (gastaldi) [inline extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java:28]: Isn't it making the List mutable?

Comment (geoand) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java:28]: Nope, that's the magic of `List.of` - it returns an immutable list

Comment (gastaldi) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java:28]: Awesome, I need to update my collections skills :)

**Review by gastaldi — APPROVED** [review]
LGTM

**Review by gsmet — APPROVED** [review]
Just be careful not changing too much of this in sections that could be subject to backports.

**Comment by geoand** [comment]
> Just be careful not changing too much of this in sections that could be subject to backports.

Good point! 
I think the way it was done was very limited and should not be a problem, but it remains to be seen :)

File: extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java (hunk 1 above)
Comment (geoand) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java:28]: It's crazy how Java didn't have these all these years...

**Pull request merged**